client/jetpack-cloud/sections/pricing/sale-banner/index.tsx

```diff
@@ -56,7 +56,7 @@ const SaleBanner: React.FC< Props > = ( { coupon } ) => {
 						<div>
 							<b>{ translate( 'End of Summer Sale!' ) }</b>
 							&nbsp;
-							{ translate( 'Get %(discount)d%% off your first year of Jetpack', {
+							{ translate( 'Get %(discount)d%% off your first year of Jetpack.', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.